sunpy/timeseries/sources/eve.py

```diff
@@ -175,15 +191,14 @@ class EVESpWxTimeSeries(GenericTimeSeries):
     @peek_show
     def peek(self, column=None, **kwargs):
         """
-        Plots the time series in a new figure. An example is shown below:
+        Plots the time series in a new figure.
 
-        ..
-            .. plot::
+        .. plot::
 
-                import sunpy.timeseries
-                from sunpy.data.sample import EVE_TIMESERIES
-                eve = sunpy.timeseries.TimeSeries(EVE_TIMESERIES, source='eve')
-                eve.peek(subplots=True)
+            import sunpy.timeseries
+            from sunpy.data.sample import EVE_TIMESERIES
+            eve = sunpy.timeseries.TimeSeries(EVE_TIMESERIES, source='eve')
+            eve.peek(subplots=True)
```

Comment:
For some reason this doesn't seem to be showing up in the built version of the [docs]( https://sunpy--5200.org.readthedocs.build/en/5200/api/sunpy.timeseries.sources.ESPTimeSeries.html#sunpy.timeseries.sources.ESPTimeSeries) like the rest

Reply:
What do you mean?

Reply:
I think we should just get rid of these plot example lines, we don't have them for other `peek()` methods.

Reply:
All of them have examples, esp is the only one without

Reply:
Can you check that each example shows up and looks nice?

Reply:
https://sunpy--5200.org.readthedocs.build/en/5200/api/sunpy.timeseries.sources.EVESpWxTimeSeries.html#sunpy.timeseries.sources.EVESpWxTimeSeries
Looks bad, the others are fine I suppose

Reply:
I think we should address this.

Reply:
The figure has too much info to fit in standard figsize, either we increase the figsize which would be inconsistent in the docs, or plot the figures with a few columns (column argument already exists), I'd say the latter approach is better

Reply:
Why not make it bigger? 

Can we also not change the subplots to use the same x axis?

Reply:
I thought making it bigger would make it inconsistent with the other plots? If not then we could do that.
> Can we also not change the subplots to use the same x axis?

Yeah, `sharex=True` argument isn't passed this would case them to use the same x axis.

Reply:
They don't have to be the same size, they display different amounts of data.